ceph/ceph.py

```diff
@@ -1465,7 +1465,7 @@ def set_internal_ip(self):
         set the internal ip of the vm which differs from floating ip
         """
         out, _ = self.exec_command(
-            cmd="/sbin/ifconfig eth0 | grep 'inet ' | awk '{ print $2}'"
+            cmd="ping -c 1 `hostname` | grep -oE '[0-9]+\\.[0-9]+\\.[0-9]+\\.[0-9]+'| head -n 1"
```

Comment:
I believe there are good chances of getting loopback address.
probably needed different approach

Reply:
@sunilkumarn417 @mkasturi18 This will return loopback address only if machines are configured with IPv6 addresses

Reply:
@sunilkumarn417 If I recall correctly, we are not having IPv6 addresses in our lab environment and on those machines it will return the desired output